src/styles/base/null.scss

```diff
@@ -0,0 +1,54 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
```

Comment:
Forbidden to use anything except `box-sizing` inside `*` selector

Reply:
it is just my template of resets. Can I use it for all projects?